src/Directory.Build.props

```diff
@@ -28,30 +17,27 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- Ensures package icon is valid -->
     <None Include="..\..\images\icon.png" Pack="true" PackagePath="\" Visible="false" />
   </ItemGroup>
-  
+
   <PropertyGroup Label="Build instructions">
     <OutputType>Library</OutputType>
     <TargetFramework>net5.0</TargetFramework>
-
-    <!-- creates a regular package and a symbols package -->
+    <!-- Creates a regular package and a symbols package -->
     <IncludeSymbols>true</IncludeSymbols>
     <!-- Creates symbol package in the new .snupkg format -->
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <!--
-      instruct the build system to embed project source files that are not tracked by the source control
+      Instruct the build system to embed project source files that are not tracked by the source control
       or imported from a source package to the generated PDB.
     -->
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <!-- Recommended: Embed symbols containing Source Link in the main file (exe/dll) -->
     <DebugType>embedded</DebugType>
-
     <!-- Will generate nuget packages for each project -->
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
```

Comment:
Why are you disabling my option to have nuget package build on compile locally?

Reply:
Right no, we have a seperate workflow-step for dotnet pack

Reply:
Yes I do know, the reason why this is set to `true` is for me to have a nuget package generated as part of my `local` build so I would not have to run `dotnet pack` from command-line.

> NOTE: for our template this should be disabled by default.

Reply:
Then I think the right solution should be `false` and then you want to make a local package and want to take a shortcut - then you can make a temporarily `true`.